esti/lakectl_test.go

```diff
@@ -64,6 +64,16 @@ func TestLakectlBasicRepoActions(t *testing.T) {
 	newStorage := storage + "/new-storage/"
 	RunCmdAndVerifyFailureWithFile(t, Lakectl()+" repo create lakefs://"+repoName+" "+newStorage, false, "lakectl_repo_create_not_unique", vars)
 
+	// Validate the --storage-id flag (currently only allowed to be empty)
+	repoNameSID := generateUniqueRepositoryName()
+	storageSID := generateUniqueStorageNamespace(repoNameSID)
+	vars = map[string]string{
+		"REPO":    repoNameSID,
+		"STORAGE": storageSID,
+		"BRANCH":  mainBranch,
+	}
+	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" repo create lakefs://"+repoNameSID+" "+storageSID+" --storage-id \"\"", false, "lakectl_repo_create", vars)
+
```

Comment:
I think it would be better to separate this into a different test.
Also, missing the failure scenario when running it with storage_id != ""

Reply:
Re the missing case, see my comment on the PR - 

> The CLI isn't failing it, but the Config validation.
> So using `RunCmdAndVerifyFailureWithFile` doesn't work with a proper golden file, since it's not a CLI error.

Can you think of a way to test this, or are you OK with having the Config tests covering this part?